server/odc-service/src/main/java/com/oceanbase/odc/metadb/connection/ConnectionSpecs.java

```diff
@@ -117,6 +117,10 @@ public static Specification<ConnectionEntity> userIdEqual(Long userId) {
         return columnEqual("visibleScope", ConnectionVisibleScope.PRIVATE).and(columnEqual("ownerId", userId));
     }
 
+    public static Specification<ConnectionEntity> usernameEqual(String username) {
```

Comment:
please add some unit cases for this new query parameter~

Reply:
ok~